hermes-console-vue/src/components/console-alert/ConsoleAlert.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+  interface ConsoleAlertProps {
+    icon?: string;
+    title?: string;
+    text: string;
+    type: 'error' | 'success' | 'warning' | 'info';
+  }
+
+  const props = defineProps<ConsoleAlertProps>();
```

Comment:
We can destructurize props

Reply:
Afaik we cannot, as we would lose reactivity then. Props value on component setup will be assigned to those variables. If props value changes, destructured values will not update.